packages/shared/src/__tests__/loadClerkJsScript.test.ts

```
@@ -12,24 +12,56 @@ jest.mock('../loadScript');
+  load: jest.fn(),
+};
+
 describe('loadClerkJsScript(options)', () => {
   const mockPublishableKey = 'pk_test_Zm9vLWJhci0xMy5jbGVyay5hY2NvdW50cy5kZXYk';
 
   beforeEach(() => {
     jest.clearAllMocks();
     (loadScript as jest.Mock).mockResolvedValue(undefined);
     document.querySelector = jest.fn().mockReturnValue(null);
+
+    delete (window as any).Clerk;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)